Feature Toggles: Create API for updating feature toggle state from the feature toggle admin page (#73022)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

Keys of alpha features to enable, separated by space.

<hr>

## [feature_management]

> The options in this section configure the experimental Feature Toggle Admin Page feature, which is enabled using the `featureToggleAdminPage` feature toggle. **Use at your own risk.**

Reviewer: I can't find any other places in the docs were we use the phrase "Use at your own risk."  Currently consulting with the team on if we have a standard caution for experimental features.

Reviewer: ```suggestion
The options in this section configure the experimental Feature Toggle Admin Page feature, which is enabled using the `featureToggleAdminPage` feature toggle. Projects and features in the experimental stage are not meant to be used in production environments, and the risks are unknown.
```

Reviewer: Another option:
```suggestion
The options in this section configure the experimental Feature Toggle Admin Page feature, which is enabled using the `featureToggleAdminPage` feature toggle. Grafana Labs offers support on a best-effort basis, and breaking changes might occur prior to the feature being made generally available.
```

Please see [Configure feature toggles]({{< relref "/feature-toggles" >}}) for more information.

### allow_editing

> Allow users to toggle feature toggle state in the feature management page. Default is `false`.

Reviewer: ```suggestion
Lets you switch the feature toggle state in the feature management page. The default is `false`.
```

### update_controller_url

> Set the URL of the controller that manages feature toggle updates. If not set, feature toggles in the feature management page will be read-only.

Reviewer: ```suggestion
Set the URL of the controller that manages the feature toggle updates. If not set, feature toggles in the feature management page will be read-only.
```

> **Note:** The API for feature toggle updates has not been defined yet; please check back later.

Reviewer: ```suggestion
{{% admonition type="note" %}}
The API for feature toggle updates has not been defined yet.
{{% /admonition %}}
```

### hidden_toggles

> Hide additional specific feature toggles from the feature management page. By default, feature toggles in stages `unknown`, `experimental`, and `private preview` are hidden from the UI. Use this option to hide toggles in stages `public preview`, `general availability`, and `deprecated`.

Reviewer: ```suggestion
Hide additional specific feature toggles from the feature management page. By default, feature toggles in the `unknown`, `experimental`, and `private preview` stages are hidden from the UI. Use this option to hide toggles in the `public preview`, `general availability`, and `deprecated` stages.
```

### read_only_toggles

> Disable updates for additional specific feature toggles in the feature management page. By default, feature feature toggles can only be updated if they are in stages `general availability` and `deprecated`. Use this option to disable updates for toggles in those stages.

Reviewer: ```suggestion
Use to disable updates for additional specific feature toggles in the feature management page. By default, feature toggles can only be updated if they are in the `general availability` and `deprecated`stages. Use this option to disable updates for toggles in those stages.
```
